--- skkinput-3.0.6/Skkinput.conf.orig	2003-01-30 17:38:00.000000000 +0100
+++ skkinput-3.0.6/Skkinput.conf	2004-04-08 18:38:13.018426176 +0200
@@ -10,7 +10,7 @@
 
 /*	 Skkinput3 Window uses these fonts. 
  */
-#define Default_FontSet -*-fixed-medium-r-normal--16-*-*-*-*-*-iso8859-1,-misc-fixed-medium-r-normal--16-*-*-*-*-*-jisx0201.1976-0,-misc-fixed-medium-r-normal--16-*-*-*-*-*-jisc6226.1978-0,-misc-fixed-medium-r-normal--16-*-*-*-*-*-jisx0208.1983-0,-misc-fixed-medium-r-normal--16-*-*-*-*-*-jisx0212.1990-0
+#define Default_FontSet -adobe-helvetica-medium-r-normal--14-*-*-*-*-*-iso8859-1,-misc-fixed-medium-r-normal--14-*-*-*-*-*-jisx0208.1983-0
 
 /*	Skkinput3 supports these protocols:
  */
